### PR TITLE
Notice in Back Office where you edit image settings

### LINKED ISSUE #9605: [BOOM-5830] Notice in FO after deleting or deactivating image format
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5830).

- _**Reporter:**_ marion_francois
- _**Created at:**_ Tue, 26 Jun 2018 15:31:11 +0200

<p>When I delete or disable the "store_default" image format in Design > Image & regenerate thumbnails<br/>
 I have a notice in FO when I try to display the stores page</p>

<p>There is the same notice with "large_default" format on categories pages in FO</p>

<p>If a format is deleted or disabled, the page should be displayed without notice & without the deleted images </p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by AlexTechDev**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           |  develop
| Description?      | This Pull Request add the improvement request in this issue https://github.com/PrestaShop/PrestaShop/issues/9605
| Type?             |  improvement 
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/9605
| How to test?      | In back office go to Design-> Image Settings -> Click on edit one image settings

image

### CODE REVIEW

**Review by atomiix — DISMISSED** [review]

**Review by Hlavtox — CHANGES_REQUESTED** [review]
I think that these messages should be in the template itself. 🤔
Notifications should be used for one-time flash messages only.

**Comment by kpodemski** [comment]
@Hlavtox legacy warnings are not really flash messages like the one you know from Symfony, I think it's fine 👍

**Review by kpodemski — DISMISSED** [review]

**Review by matks — COMMENTED** [review]
Waiting for @Julievrz validation of the wording

File: controllers/admin/AdminImagesController.php [hunk 1]
```diff
@@ -774,6 +774,7 @@ public function initContent()
 
         if ($this->display == 'edit') {
             $this->warnings[] = $this->trans('After modification, do not forget to regenerate thumbnails', [], 'Admin.Design.Notification');
+            $this->warnings[] = $this->trans('If you delete image format, make sure the theme you use does not rely on it', [], 'Admin.Design.Notification');
```
Comment (Julievrz) [inline controllers/admin/AdminImagesController.php]:
```suggestion
            $this->warnings[] = $this->trans('Make sure the theme you use doesn't rely on this image format before deleting it.', [], 'Admin.Design.Notification');
```

Comment (AlexTechDev) [reply controllers/admin/AdminImagesController.php]: Sounds good!
Thanks!

**Comment by kpodemski** [comment]
Hi @AlexTechDev 

Could you commit Julie's suggestion? Make sure to escape `doesn't` properly :)

**Comment by AlexTechDev** [comment]
> Hi @AlexTechDev
> 
> Could you commit Julie's suggestion? Make sure to escape `doesn't` properly :)

Done! :)

**Review by AlexTechDev — COMMENTED** [review]
Doesn't --> does not

File: controllers/admin/AdminImagesController.php [hunk 2]
```diff
@@ -774,6 +774,7 @@ public function initContent()
 
         if ($this->display == 'edit') {
             $this->warnings[] = $this->trans('After modification, do not forget to regenerate thumbnails', [], 'Admin.Design.Notification');
+            $this->warnings[] = $this->trans('Make sure the theme you use does not rely on this image format before deleting it.', [], 'Admin.Design.Notification');
```
Comment (kpodemski) [inline controllers/admin/AdminImagesController.php]:
```suggestion
            $this->warnings[] = $this->trans('Make sure the theme you use doesn\'t rely on this image format before deleting it.', [], 'Admin.Design.Notification');
```

Comment (kpodemski) [reply controllers/admin/AdminImagesController.php]: let's keep it as Julie suggested :)

Comment (Julievrz) [reply controllers/admin/AdminImagesController.php]: Hi, thanks. We use contractions on the project to give a human touch to wordings. :)

**Comment by kpodemski** [comment]
@AlexTechDev

**Comment by AlexTechDev** [comment]
Fine for me.
Sorry I don't understand you correctly

**Review by kpodemski — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by khouloudbelguith — APPROVED** [review]
Hi @AlexTechDev,

It is ok :heavy_check_mark: 
Automatic tests => OK
I tried to edit an image with multistore enabled & disabled => ok
I tried with employee language = English & French
I tried with a mobile => OK
image

https://user-images.githubusercontent.com/16067358/194565826-35ed550e-9e52-4e21-a010-04614d2577ad.mp4

Thanks!

**Comment by atomiix** [comment]
Thanks @AlexTechDev

**Pull request merged**